src/coreimage.cs

```diff
@@ -1841,6 +1880,62 @@ interface CIImage : NSSecureCoding, NSCopying {
 		[Mac (10,13)]
 		[Export ("imageTransformForCGOrientation:")]
 		CGAffineTransform GetImageTransform (CGImagePropertyOrientation orientation);
+
+		[TV (12, 0), iOS (12, 0), Mac (10, 14, onlyOn64: true)]
+		[Export ("imageByInsertingIntermediate")]
+		CIImage CreateByInsertingIntermediate ();
+
+		[TV (12, 0), iOS (12, 0), Mac (10, 14, onlyOn64: true)]
+		[Export ("imageByInsertingIntermediate:")]
+		CIImage CreateByInsertingIntermediate (bool cache);
+
+		// CIImage_AVPortraitEffectsMatte category
+
+		[TV (12, 0), iOS (12, 0), Mac (10, 14, onlyOn64: true)]
+		[NullAllowed, Export ("portraitEffectsMatte")]
+		AVPortraitEffectsMatte PortraitEffectsMatte { get; }
+
+		[TV (12,0), iOS (12,0), Mac (10, 14, onlyOn64: true)]
+		[Export ("initWithPortaitEffectsMatte:options:")] // selector typo, rdar filled 42894821
+		IntPtr Constructor (AVPortraitEffectsMatte matte, [NullAllowed] NSDictionary<NSString, NSObject> options);
+
+		[TV (11,0), iOS (11,0), Mac (10, 14, onlyOn64: true)]
+		[Export ("initWithPortaitEffectsMatte:")] // selector typo, rdar filled 42894821
+		IntPtr Constructor (AVPortraitEffectsMatte matte);
+
+		[TV (12,0), iOS (12,0), Mac (10, 14, onlyOn64: true)]
+		[Static]
+		[Export ("imageWithPortaitEffectsMatte:options:")] // selector typo, rdar filled 42894821
+		[return: NullAllowed]
+		CIImage FromPortraitEffectsMatte (AVPortraitEffectsMatte matte, [NullAllowed] NSDictionary<NSString, NSObject> options);
```

Comment:
use the non-generic `NSDictionary` since that signature is not very helpful and it will make it harder to create strong dictionaries in the future